packages/library-base/src/commonMain/kotlin/io/realm/kotlin/query/RealmQuery.kt

```diff
@@ -23,6 +23,8 @@ import io.realm.kotlin.types.RealmList
 import kotlinx.coroutines.flow.Flow
 import kotlin.reflect.KClass
 
+public const val TRUE_PREDICATE: String = "TRUEPREDICATE"
```

Comment:
Nitpick, should we rename this to `ALWAYS_TRUE` to better signal what it does? I always found `true_predicate` pretty hard to reason about...but not a big deal.

Reply:
As a public property this should also have documentation